src/sdk/pynni/nni/msg_dispatcher_base.py

```diff
@@ -168,6 +169,9 @@ def handle_initialize(self, data):
     def handle_request_trial_jobs(self, data):
         raise NotImplementedError('handle_request_trial_jobs not implemented')
 
+    def handle_resubmit_trial_job(self, data):
+        raise NotImplementedError('handle_resubmit_trial_job not implemented')
+
     def handle_update_search_space(self, data):
        raise NotImplementedError('handle_update_search_space not implemented')
```

Comment:
Irrelevant to this PR. The indentation is mistyped.

Reply:
why irrelevant ? Which indentation ?

Reply:
fixed